### DISCUSSION TITLE
TypeError: Checker.return_text() missing 1 required positional argument: 'locator_value'

### DISCUSSION THREAD

**Discussion by SachinSoni4619** [NetzOptimize/pyQA#14] [Q&A] [answered]
g = run.return_text("//a[normalize-space()='Parking']")

This keeps giving error :
TypeError: Checker.return_text() missing 1 required positional argument: 'locator_value'

**Comment by Montekkundan** [comment, marked as answer]
You have to add a selector (xpath, css etc) :

Try doing this:
g = run.return_text("xpath", "//a[normalize-space()='Parking']")
